database/migrations/2014_10_12_001234_create_followers_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateFollowersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('followers', function (Blueprint $table) {
+            // $table->increments('id');
```

Comment:
on the er diagram there are only 2 attributes which is users_id and followers_user_id,  but i commented it out because i might need in the future if i start implementing follow function.